### PR TITLE
doc/rados: fixed hit set type link

### PR DESCRIPTION
**Pull request by falcon78921**
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

Fixes: https://tracker.ceph.com/issues/34539
Signed-off-by: James McClune jmcclune@mcclunetechnologies.net

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins render docs

**Comment by falcon78921** [comment]
jenkins render docs

File: doc/rados/configuration/mon-config-ref.rst [hunk 1]
```diff
@@ -348,8 +348,8 @@ by setting it in the ``[mon]`` section of the configuration file.
 ``mon warn on cache pools without hit sets``
 
 :Description: Issue a ``HEALTH_WARN`` in cluster log if a cache pool does not
-              have the hitset type set set.
-              See `hit set type <../operations/pools#hit-set-type>`_ for more
+              have the ``hit set type`` value configured.
+              See :ref:`hit set type <_hit_set_type>` for more
```
[the remark this answers is not in the document]
Comment (falcon78921) [reply doc/rados/configuration/mon-config-ref.rst]: @tchaikov Oops, the label was ``hit_set_type``. I had an extra underscore.

Could you please review again, when you have the chance?

**Comment by falcon78921** [comment]
jenkins render docs

**Pull request merged**